reference/config_files/editable_layout.rst

```diff
@@ -33,9 +40,13 @@ This file can live in the conan cache, in the ``.conan/layouts`` folder, or in a
 inside the source repo.
 
 If there exists a ``.conan/layouts/default`` layout file in the cache and no layout file is specified
-in the ``conan link <path> <reference>`` command, that file will be used.
+in the ``conan editable add <path> <reference>`` command, that file will be used.
```

Comment:
That file (`.conan/layouts/default`) will no longer be used by default

Reply:
Yes, that has not changed. If you don't specify a layout, that file will be used by default.